roda-core/roda-core/src/main/java/org/roda/core/plugins/base/notifications/EmailIngestNotification.java

```diff
@@ -94,13 +95,16 @@ public void notify(ModelService model, IndexService index, Job job, JobStats job
             StringBuilder builder = new StringBuilder();
 
             for (IndexedReport report : reports) {
-              Report last = report.getReports().get(report.getReports().size() - 1);
+              List<Report> reportsList = RodaCoreFactory.getModelService().retrieveJobReport(report.getJobId(), report.getId()).getReports();
```

Comment:
There is no need for RodaCoreFactory.getModelService() you already have the `model` object as a parameter